packages/stargate/src/queries/ibc.spec.ts

```diff
@@ -125,12 +130,8 @@ describe("IbcExtension", () => {
           1,
           2,
           3,
-          4,
-          5,
-          6,
-          7,
         ]);
-        expect(response.sequences).toEqual([4, 5, 6, 7].map((n) => Long.fromInt(n, true)));
+        expect(response.sequences).toEqual([1, 2, 3].map((n) => Long.fromInt(n, true)));
```

Comment:
This test got bording. We "unreceived" all of the requested packages

Reply:
bording? broken?

I will look in more. If it is an error in the relayer/genesis, or in our code, or in the Cosmos SDK :grimacing:

Reply:
"Boring", sorry.